tests/OSPSuite.Starter/Presenters/MenuAndToolBarPresenter.cs

```diff
@@ -26,18 +26,128 @@ protected override void DisableMenuBarItemsForPogramStart()
       protected override void AddRibbonPages()
       {
          _view.AddPageGroupToPage(createGroupModelling(), "Modeling");
-         _view.AddPageGroupToPage(createParmaeterIdentification(), "Modeling");
+         _view.AddPageGroupToPage(createParameterIdentification(), "Modeling");
+         _view.AddPageGroupToPage(createParameterIdentificationAnalyses(), "Modeling");
+         _view.AddPageGroupToPage(createParameterIdentificationConfidenceIntervals(), "Modeling");
+
+//         _view.AddPageGroupToPage(createSensitivityAnalysis(), "Modeling");
```

Comment:
Going to add this next. First it has to be possible to run the sensitivity analysis